src/main/java/io/vinyldns/java/VinylDNSClient.java

```diff
@@ -358,4 +358,17 @@
    *     case of failure
    */
   VinylDNSResponse<BatchResponse> cancelBatchChanges(String id);
+
+  /**
+   * Retrieves a list of RecordSets globally in the VinylDNS database based on search criteria. A
+   * minimum of two alpha-numeric characters is required.
+   *
+   * @param request See {@link ListRecordSetGlobalRequest ListRecordSetGlobalRequest Model}
+   * @return {@link VinylDNSSuccessResponse
+   *     VinylDNSSuccessResponse&lt;ListRecordSetGlobalResponse&gt;} in case of success and {@link
+   *     VinylDNSFailureResponse VinylDNSFailureResponse&lt;ListRecordSetGlobalResponse&gt;} in case
+   *     of failure
+   */
+  VinylDNSResponse<ListRecordSetGlobalResponse> listGlobalRecordSets(
```

Comment:
This is all good, just naming.

I would call this "search" to disambiguate it from the existing `listRecordSet`

- `SearchRecordSetsRequest`
- `SearchRecordSetsResponse`
- `public VinylDNSResponse<SearchRecordSetsResponse> searchRecordSets(SearchRecordSetsRequest request)...`

Thanks again for doing this, looks great!

Reply:
Renamed :)